projects/exui-common-lib/src/lib/models/basic-user.model.ts

```diff
@@ -1,4 +1,5 @@
 export interface BasicUser {
+    [key: string]: any;
```

Comment:
Is this needed? it opens up the interface to have any properties

Reply:
not added by me, but removed now.